json_serializable/lib/src/type_helpers/json_converter_helper.dart

```diff
@@ -49,6 +67,24 @@ class JsonConverterHelper extends TypeHelper {
 
     final asContent = asStatement(converter.jsonType);
 
+    if (!converter.jsonType.isNullableType && targetType.isNullableType) {
+      const converterFromJsonName = r'_$JsonConverterFromJson';
```

Comment:
Uh....what if there are multiple different converters used in the same library?

Reply:
That's fine because the helper is independent from the converter. The helper is a bit more complex than necessary (generic helper with a callback parameter) to make it reusable between converters.

That's the only way I found to prevent name clash and handle fancy cases like:

```dart
const converter = MyConverter();

...
@converter
final int field;
```

Reply:
Riiiiight!